docs/ref/models/fields.txt

```diff
@@ -12,11 +12,14 @@ This document contains all the API references of :class:`Field` including the
 
 .. seealso::
 
-    If the built-in fields don't do the trick, you can try `django-localflavor
-    <https://django-localflavor.readthedocs.org/>`_, which contains assorted
-    pieces of code that are useful for particular countries or cultures. Also,
-    you can easily :doc:`write your own custom model fields
-    </howto/custom-model-fields>`.
+   If the built-in fields don't do the trick, you can refer to the
```

Comment:
4 space indent please
I think 'try' as originally worded is a better word choice than 'refer to'.